src/components/AutoComplete/AutoComplete.stories.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-alert */
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Row, Col } from '../../index';
+
+import AutoComplete from './AutoComplete';
+import { getSearchItems } from './AutoComplete.fixtures';
+
+export class AutoCompleteExample extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { items: [] };
+  }
+
+  componentDidMount() {
+    this.onInputUpdate();
+  }
+
+  onInputUpdate = (searchTerm = '') => {
+    searchTerm = searchTerm.trimLeft();
+
+    this.setState({
+      items: getSearchItems(searchTerm)
+    });
+  };
+
+  render() {
+    return (
+      <AutoComplete
+        items={this.state.items}
+        labelText="Type your favorite color"
+        onInputUpdate={this.onInputUpdate}
+        onSearch={selection => alert(`You selected ${selection}!`)}
+      />
+    );
+  }
+}
+
+storiesOf('AutoComplete', module)
+  .addDecorator(getStory => (
+    <div style={{ padding: 20 }}>
+      <Row>
+        <Col sm={3} />
+        <Col sm={6}>{getStory()}</Col>
+      </Row>
+      <Row>
+        <Col sm={3} />
+        <Col sm={6}>
+          <p>The dropdown should overlay this text.</p>
+        </Col>
+      </Row>
+    </div>
+  ))
```

Comment:
Really need this decorator?

Reply:
The main purpose of this is to show how the dropdown will stick to the input field regardless of where it is on the screen.